playbooks/install_stack.yaml

```diff
@@ -387,6 +391,27 @@
       - sriov_interface is defined
       - not ovn_enabled
 
+  - name: Add dpdk-ovn to enabled services
+    set_fact:
+      service_envs: "{{ service_envs | union(dpdk_env) }}"
+    vars:
+      dpdk_env:
+        - /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovn-dpdk.yaml
+    when:
+    - dpdk_interface is defined
+    - ovn_enabled
+
+  - name: Add dpdk-ovs to enabled services
+    set_fact:
+      service_envs: "{{ service_envs | union(dpdk_env) }}"
+    vars:
+      dpdk_env:
+        - /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovs.yaml
```

Comment:
Is the `neutron-ovs.yaml` environment file something that we should conditionally import based on the RHOSP version?

Why don't we have a similar environment file for neutron-ovn?

Reply:
because OVN is enabled by default in TripleO starting from OSP 16; so we don't have anything to import by default.